web/packages/teleterm/src/ui/ConnectMyComputer/CompatibilityPromise.tsx

```diff
@@ -26,22 +26,24 @@ import { RuntimeSettings } from 'teleterm/mainProcess/types';
 const CONNECT_MY_COMPUTER_RELEASE_VERSION = '14.1.0';
 const CONNECT_MY_COMPUTER_RELEASE_MAJOR_VERSION = 14;
 
-export function isAgentCompatible(
+export type AgentCompatibility = 'unknown' | 'compatible' | 'incompatible';
+
+export function checkAgentCompatibility(
   proxyVersion: string,
   runtimeSettings: Pick<RuntimeSettings, 'appVersion' | 'isLocalBuild'>
-): boolean {
-  if (proxyVersion === '') {
-    return false;
+): AgentCompatibility {
+  if (!proxyVersion) {
```

Comment:
Let's leave a comment explaining why and when this might happen.

Reply:
Done.